### PR TITLE
post: ng v11

### PR DESCRIPTION
**Pull request by cexbrayat**

### CODE REVIEW

File: _posts/2020-11-11-angular-cli-11.0.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+As a result, very few people used it.
+
+It's now as simple as running `ng serve --hmr` 😍.
+You can now enjoy tweaking your components
+without losing the state of your application on the live-reload.
+
+### Custom headers
+
+It is now possible for `ng serve` to send custom HTTP headers.
+This is part of the effort to support Trusted types in Angular,
```
Comment (jnizet) [inline _posts/2020-11-11-angular-cli-11.0.md:69]: If you have a link explaining what trusted types and these headers are for, that would be nice here.

File: _posts/2020-11-11-angular-cli-11.0.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+- before: `"styles": ["../node_modules/bootstrap/dist/css/bootstrap.css"]`
+- after: `"styles": ["bootstrap/dist/css/bootstrap.css"]`
+
+The `extractCss` has now a default value of `true` and has been deprecated:
+you can remove it from your `angular.json` config
+(or let the schematic do it for you when upgrading).
+This means we no longer have a `styles.js` when building the application.
+
+Some other options previously deprecated have been removed,
+so you probably don't have them anymore.
```
Comment (jnizet) [inline _posts/2020-11-11-angular-cli-11.0.md:96]: Some other options previously deprecated have been removed,.
You probably don't use them anymore, but if you still do, the migration schematic will replace them with the proper alternative.

File: _posts/2020-11-11-angular-cli-11.0.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+## ng generate
+
+We can now generate resolvers \o/
+
+    ng generate resolver users
+
+The command generates a resolver and its test.
+You can also use `ng g r users`.
+
+Not that the `--lint-fix` flag the `generate` command offered has been deprecated:
```
Comment (jnizet) [inline _posts/2020-11-11-angular-cli-11.0.md:132]: Note that the `--lint-fix` flag of the `generate` command has been deprecated:

File: _posts/2020-11-11-what-is-new-angular-11.0.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+but they'll probably indicate real issues with your application.
+
+## Forms
+
+The base class for form controls, groups, and arrays has several properties like `invalid`, `dirty`, etc.
+It also has a property `parent`, allowing to get the parent form group or array.
+The type returned by `parent` has been updated from `FormGroup | FormArray` to `FormGroup | FormArray | null` as it can indeed return `null` (for example for the root form group).
+This should not impact you, but if you use `control.parent.something` somewhere,
+you now have to take care of the potential null value of `parent`.
+As often, the Angular team already wrote a migration for us.
```
Comment (jnizet) [inline _posts/2020-11-11-what-is-new-angular-11.0.md:157]: Don't you only have to deal with it if the "strict null checks" Typescript option is on?

File: _posts/2020-11-11-what-is-new-angular-11.0.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+    </trans-unit>
+    <trans-unit id="5e3335d7f1a430ef14a91507531838c57138b7f2" datatype="html">
+      <source>Welcome</source>
+      <context-group purpose="location">
+        <context context-type="sourcefile">src/app/app.component.html</context>
+        <context context-type="linenumber">2</context>
+      </context-group>
+    </trans-unit>
+
+The legacy method is also not very good at handling spaces in ICUs expressions for example.
```
Comment (jnizet) [inline _posts/2020-11-11-what-is-new-angular-11.0.md:191]: ICU expressions

**Review by jnizet — CHANGES_REQUESTED** [review]

**Comment by jnizet** [comment]
Great posts! 👏

File: _posts/2020-11-11-what-is-new-angular-11.0.md (hunk 4 above)
Comment (cexbrayat) [reply _posts/2020-11-11-what-is-new-angular-11.0.md:157]: Yeah probably 👍

**Pull request merged**